src/rpc/blockchain.cpp

```diff
@@ -2058,6 +2058,96 @@ UniValue scantxoutset(const JSONRPCRequest& request)
     return result;
 }
 
+static UniValue getcolor(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() < 2 || request.params.size() > 3)
+        throw std::runtime_error(
+            "getcolor \"token_type\" \"txid/scriptpubkey\" index \n"
+            "\nGet the color of the token that can be generated when using the script pubkey given as parameter or transaction id and index pair given as parameter.\n"
+            "\nArguments:\n"
+            "1. \"token_type\"       (numberic, required) Value can be 1 or 2 or 3.\n"
+            " 1. REISSUABLE\n"
+            " 2. NON-REISSUABLE\n"
+            " 3. NFT\n"
+            "2. \"scriptpubkey\"     (string, optional) Script pubkey that is used to issue REISSUABLE token\n"
+            "2. \"txid\"             (string, optional) Transaction id from which the NON-REISSUABLE or NFT tokens are issued\n"
+            "3. \"index\"            (numeric, optional) Index in the above transaction id used for issuing token\n"
+            "\nResult:\n"
+            "\"color\"               (string) The color or token.\n"
+            "\nExamples:\n"
+            + HelpExampleCli("getcolor", "\"1\" 8282263212c609d9ea2a6e3e172de238d8c39cabd5ac1ca10646e23fd5f51508")
+            + HelpExampleCli("getcolor", "\"2\" 485273f6703f038a234400edadb543eb44b4af5372e8b207990beebc386e7954 0")
+            + HelpExampleCli("getcolor", "\"3\" 485273f6703f038a234400edadb543eb44b4af5372e8b207990beebc386e7954 1")
+        );
```

Comment:
All errors occur when the example are executed.

```
$ ./src/tapyrus-cli getcolor "1" 8282263212c609d9ea2a6e3e172de238d8c39cabd5ac1ca10646e23fd5f51508
error code: -8
error message:
Invalid Tapyrus script
$ ./src/tapyrus-cli getcolor "2" 485273f6703f038a234400edadb543eb44b4af5372e8b207990beebc386e7954 0
error code: -8
error message:
Invalid transaction id :485273f6703f038a234400edadb543eb44b4af5372e8b207990beebc386e7954
$ ./src/tapyrus-cli getcolor "3" 485273f6703f038a234400edadb543eb44b4af5372e8b207990beebc386e7954 1
error code: -8
error message:
Invalid transaction id :485273f6703f038a234400edadb543eb44b4af5372e8b207990beebc386e7954
```

Reply:
In the first case I'll change the script. 
But the other two can't be fixed. Giving a valid transaction id is not possible, we can only have a hex string of the correct size. It can't be guaranteed to be valid in the environment where it is executed.

Reply:
I think so too. In addition to that same data as in the example should be added to the test case.